Fastfile

```diff
@@ -14,6 +14,15 @@ default_platform :ios
     scan(skip_slack: skip_slack)
   end
 
+  lane :ui_test do
+    _setup()
+    skip_slack = ENV['SCAN_SLACK_CHANNEL'].to_s.strip.empty?
+    scan(skip_slack: skip_slack,
+         devices: ['TAB_UI_TEST_DEVICES'],
+         output_types: ['TAB_OUTPUT_TYPES'],
```

Comment:
Could this be a more descriptive name, Output Types could refer to a lot of different things